src/components/vedlegg/EttersendelseView.tsx

```diff
@@ -146,7 +146,7 @@ const EttersendelseView: React.FC<Props> = ({restStatus}) => {
     return (
         <>
             <DriftsmeldingVedlegg
-                restStatusSjekk={restStatus === REST_STATUS.INITIALISERT || restStatus === REST_STATUS.PENDING}
+                restStatusError={restStatus !== REST_STATUS.INITIALISERT && restStatus !== REST_STATUS.PENDING}
```

Comment:
Her blir det også feil å bruke denne varianten, da sjekken restStatusSjekk={restStatus === REST_STATUS.INITIALISERT || restStatus === REST_STATUS.PENDING} egentlig sjekker på isLoading, men komponenten skal ta i mot hasError

Reply:
Tror egentlig denne bare kan være som den var?
For her var det riktig at leserData skulle bli sendt videre? Må isåfall også endre tilbake i DriftsmelidngVedlegg

Reply:
Ok, tar å reversere det.